### PR TITLE
Google search

### PR DESCRIPTION
**Pull request by eugeniostreliaev**

### CODE REVIEW

**Comment by eugeniostreliaev** [comment]
obrigado !!

File: index.js [hunk 1]
```diff
@@ -1,32 +1,18 @@
...
-      const selectedPrefixIndex = readline.keyInSelect(prefixes, 'Choose an option: ')
-      const selectedPrefixText = prefixes[selectedPrefixIndex]
-      return selectedPrefixText
-  }
-
-   console.log(JSON.stringify(content, null, 10))
-  //console.log(content)
+  robots.input()
+  await robots.text()
+  await robots.image()
```
Comment (gracietti) [inline index.js:12]: How many robots!! The squad is getting huge! 🤖 🤖 🤖

File: package.json [hunk 2]
```diff
@@ -18,6 +18,7 @@
   "homepage": "https://github.com/eugeniostreliaev/video-maker#readme",
   "dependencies": {
     "algorithmia": "^0.3.10",
+    "googleapis": "^40.0.0",
```
Comment (gracietti) [inline package.json:21]: Google and IBM... who is playing with world's top companies API? 🎖

File: robots/image.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    const content = state.load()
+    await fetchImagesOfAllSentences(content)
+    state.save(content)
+
+    async function fetchImagesOfAllSentences(content) {
+        for (const sentence of content.sentences) {
+            const query = `${content.searchTerm} ${sentence.keywords[0]}`
+            sentence.images = await fetchGoogleAndReturnImagesLinks(query)
+
+            sentence.googleSearchQuery = query
```
Comment (gracietti) [inline robots/image.js:17]: Why do you need to save query on googleSearchQuery?

File: robots/image.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    }
+
+
+    async function fetchGoogleAndReturnImagesLinks(query) {
+        const response = await customSearch.cse.list({
+            auth: googleSearchCredentials.apiKey,
+            cx: googleSearchCredentials.searchEngineId,
+            q: query,
+            searchType: 'image',
+            num: 2
```
Comment (gracietti) [inline robots/image.js:28]: You could add the parameter `imgSize: xlarge` to assure you'll get images with good enough quality, but not too big

https://developers.google.com/custom-search/v1/cse/list

File: robots/image.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+        })
+
+
+        const imagesUrl = response.data.items.map((item) => {
```
Comment (gracietti) [inline robots/image.js:32]: Beautiful parsing 👍

File: robots/text.js [hunk 6]
```diff
@@ -72,17 +82,23 @@ async function robot(content) {
...
 
+
     function limitMaximumSentences(content) {
         content.sentences = content.sentences.slice(0, content.maximumSentences)
     }
 
+
+
     async function fetchKeywordsOfAllSentences(content) {
+        console.log('Magically fetching keywords from Watson. Wait and see...')
```
Comment (gracietti) [inline robots/text.js:93]: 😄  😮

File: robots/text.js [hunk 7]
```diff
@@ -72,17 +82,23 @@ async function robot(content) {
...
         for (const sentence of content.sentences) {
             sentence.keywords = await fetchWatsonAndReturnKeywords(sentence.text)
         }
     }
 
+
     // receives a sentence as parameter and returns keywords to be used as please
+    // the sentence enters into an object that is then sentt to watson
```
Comment (gracietti) [inline robots/text.js:101]: Vamo logo deitar abraçadinho!! 👨‍❤️‍👨  🛏

**Review by gracietti — APPROVED** [review]
Well done!! 🏅

**Pull request merged**